declaration.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
+  const content: string;
```

Comment:
```suggestion
  const content: React.FunctionComponent<React.SVGAttributes>;
```
svg를 컴포넌트화 해서 사용할 경우 이런한 형태의 type을 넣을 수 있답니다.